Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoft365Defender.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: M365 Defender Network Sessions
-  Version: '0.1'
-  LastUpdated: Aug 4, 2021
+  Version: '0.2'
```

Comment:
Although no direct change here.

Reply:
As said on the web parsers. Point taken, but for this iteration I will keep this like this.